source/mqtt_client_connection.c

```diff
@@ -1806,3 +1806,86 @@ napi_value aws_napi_mqtt_client_connection_disconnect(napi_env env, napi_callbac
 
     return NULL;
 }
+
+static int s_create_napi_mqtt_connection_statistics(
+    napi_env env,
+    const struct aws_mqtt_connection_operation_statistics *stats,
+    napi_value *stats_out) {
+
+    if (env == NULL) {
+        return aws_raise_error(AWS_CRT_NODEJS_ERROR_THREADSAFE_FUNCTION_NULL_NAPI_ENV);
+    }
+
+    napi_value napi_stats = NULL;
+    AWS_NAPI_CALL(
+        env, napi_create_object(env, &napi_stats), { return aws_raise_error(AWS_CRT_NODEJS_ERROR_NAPI_FAILURE); });
+
+    if (aws_napi_attach_object_property_u64(
+            napi_stats, env, "incompleteOperationCount", stats->incomplete_operation_count)) {
```

Comment:
Let's make the string constants actual global constants similar to what we do in mqtt5_client.c